app/src/pages/CounterPage/containers/CounterPageContainer.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import CounterLayout from "../components/CounterLayout";
+
+class CounterPageConteiner extends Component{
+    constructor(props){
+        super(props);
+        this.state={
+            countValue: 0,
+            parityType: 'even',
+            filling:{
+                background:'white'
+            }
+        }
+    };
+    handleIncrement = () =>{
+        this.setState({countValue: this.state.countValue +1});
+        if(this.state.countValue % 2 === 0){
+            this.state.parityType = 'odd';
+            this.state.filling = {background :'red'}
```

Comment:
here you mutate state directly. It must be inside setState method. Use callback inside setState and add it there